src/MediatR/Internal/RequestHandlerWrapper.cs

```diff
@@ -1,3 +1,5 @@
+using MediatR.Pipeline;
```

Comment:
Using in a different place?

Reply:
This is due to us referencing `PipelinePriorityAttribute`. Do you prefer it be in a different namespace? Remember we can't put it in the  `Internal` namespace because it needs to be used by others as well.